### ISSUE TITLE
Lock all settings -except for camera- to specific camera.

### ISSUE THREAD

**Issue by PlasticTasticc** [chunky-dev/chunky#1395]
As a sort of Chunky equivelant to the plugin for 3DS Max "RPM
![CommonParamsExpandedGlobals](https://user-images.githubusercontent.com/110137296/181385547-ac4c8008-92b6-4196-b545-15e6cf477738.gif)
"
Not with that specific menu, but used as an example to illustrate the things you'd be able to do. Ie: locking resolution to one camera and having another resolution for another camera. Or one light setting for one camera, and another for anoth-... you get the idea.

Having all rollout contents, except for the camera rollout, be locked to whichever camera is selected.

**Comment by NotStirred** [comment]
Are you saying you want to have all of the scene settings per-camera, so that if you switch camera it loads that camera's scene settings?

**Comment by PlasticTasticc** [comment]
> 

Yes, well except for the loaded map obviously. But everything else essentially yea.
